test/test-applications/integrations/dependency-libs/TestLibrary.InstrumentationTarget/TestLibrary.InstrumentationTarget.csproj

```diff
@@ -5,4 +5,8 @@
     <AddTestStrongNameAssemblyKeyOnNetFramework>true</AddTestStrongNameAssemblyKeyOnNetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\..\src\OpenTelemetry.AutoInstrumentation\OpenTelemetry.AutoInstrumentation.csproj" />
```

Comment:
I don't like to bring this reference: it can mask problems because we bring all the dependencies. That said it facilitates things here. Could you please do a manual run in which you delete de assemblies brought by this project reference?

Reply:
All tests for .NET Framework passed 
![image](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/assets/5972917/4ac8af60-9da2-42c9-bc5d-3f4471836473)

`IntegrationTest.StrongNamedTests` for .NET 6.0 and .NET 7.0 targets are failing due to. I would say that it is expected.
```log
Xunit.Sdk.XunitException
Expected process.ExitCode to be 0 because Test application exited with non-zero exit code, but found -2147450740 (difference of -2147450740).
   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Numeric.NumericAssertions`2.Be(T expected, String because, Object[] becauseArgs)
   at IntegrationTests.Helpers.TestHelper.RunTestApplication(TestSettings testSettings) in C:\GitHub\opentelemetry-dotnet-instrumentation\test\IntegrationTests\Helpers\TestHelper.cs:line 126
   at IntegrationTests.StrongNamedTests.SubmitsTraces() in C:\GitHub\opentelemetry-dotnet-instrumentation\test\IntegrationTests\StrongNamedTests.cs:line 43
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)

Found profiler at C:\GitHub\opentelemetry-dotnet-instrumentation\bin\tracer-home\win-x64\OpenTelemetry.AutoInstrumentation.Native.dll.
Platform: X64
Configuration: Debug
TargetFramework: net7.0
.NET Core: True
Profiler DLL: C:\GitHub\opentelemetry-dotnet-instrumentation\bin\tracer-home\win-x64\OpenTelemetry.AutoInstrumentation.Native.dll
[TestHttpServer]: Listening on 'http://127.0.0.1:51124//v1/traces'
Starting Application: C:\GitHub\opentelemetry-dotnet-instrumentation\test\test-applications\integrations\bin\TestApplication.StrongNamed\x64\Debug\net7.0\TestApplication.StrongNamed.dll
ProcessName: dotnet
ProcessId: 14232
Exit Code: -2147450740
StandardError:
Error:
  An assembly specified in the application dependencies manifest (TestApplication.StrongNamed.deps.json) was not found:
    package: 'OpenTelemetry.AutoInstrumentation.Runtime.Managed', version: '0.7.0'
    path: 'OpenTelemetry.AutoInstrumentation.dll'

[MockSpansCollector]: Shutting down.
[TestHttpServer]: Shutting down

```